.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -183,12 +164,14 @@ jobs:
       - name: Cypress run
         run: |
           cd ui
+          npm install cypress --save-dev
           npx cypress run
 
       - uses: actions/setup-go@v2
         with:
-          go-version: "1.24.3"
+          go-version: "1.23.0"
```

Comment:
Why are we decreasing go version?

Reply:
Changing this to match with the the Build version set in go.mod.